src/Uno.UI/UI/Xaml/FrameworkElement.cs

```diff
@@ -794,8 +794,7 @@ private bool IsHeightConstrained(View requester)
 			return this.IsHeightConstrainedSimple();
 		}
 
-		internal override bool IsViewHit()
-			=> Background != null;
+		internal override bool IsViewHit() => false;
```

Comment:
Won't this change a lot of the hit testing behavior ?

Reply:
Not necessarily "a lot" since at the end most of cases are ending in the new `Border.IsViewHitImpl`, but it will for sure break some stretch cases. If we are about to change that, we should make sure to implement the right / final behavior.

Reply:
hopefully, yes

Reply:
This is what WinUI does. It mainly applies to `Control` which should not itself block touch, regardless of what Control.Background is set to. (Note that `Background` is a property of `Panel`, `Border`, `Control` etc on WinUI; other FrameworkElements don't have a `Background`.)

Reply:
If it's the WinUI's behavior that's perfect then. I would however add a feature flag for that (using the new fixed behavior by default). There are probably some custom controls in apps that are built assuming that settings the `Background` would do the trick (it was a common quick patch fix recommended to lot of people).

Reply:
makes sense - I added the feature flag